addons/project/models/project_task.py

```diff
@@ -864,6 +851,19 @@ def copy(self, default=None):
                 for task_id in dependant_ids
             ]
 
+    def _portal_get_parent_hash_token(self, pid):
+        return self.project_id._sign_token(pid)
```

Comment:
Nitpick, but could you move this back above line 839, to minimize noise in the diff? 😇